webcompat/static/js/lib/issues.js

```diff
@@ -426,17 +426,16 @@ issues.MainView = Backbone.View.extend({
 
       // If there are any comments, go fetch the model data
       if (this.issue.get('commentNumber') > 0) {
-        this.comments.fetch(headersBag).success(_.bind(function() {
+        this.comments.fetch(headersBag).success(_.bind(function(response) {
           this.addExistingComments();
-          // the add event is fired when a model is added to the collection.
           this.comments.bind('add', _.bind(this.addComment, this));
-
           // If there's a #hash pointing to a comment (or elsewhere)
           // scrollTo it.
           if (location.hash !== '') {
             var _id = $(location.hash);
             window.scrollTo(0, _id.offset().top);
           }
+          (response[0].lastPageNumber > 1) ? this.getRemainingComments(++response[0].lastPageNumber) : '';
```

Comment:
This kind of ternary is a bit strange. If the conditional is false, it leaving an empty string in the middle of program. That's legal... but weird.

Why not just 
```
if (foo > 1)  {
 doThing()
}
```
?